src/policy/fees.cpp

```diff
@@ -339,7 +329,7 @@ double TxConfirmStats::EstimateMedianVal(int confTarget, double sufficientTxVal,
     }
 
     LogPrint(BCLog::ESTIMATEFEE, "FeeEst: %d %s%.0f%% decay %.5f: feerate: %g from (%g - %g) %.2f%% %.1f/(%.1f %d mem %.1f out) Fail: (%g - %g) %.2f%% %.1f/(%.1f %d mem %.1f out)\n",
-             confTarget, requireGreater ? ">" : "<", 100.0 * successBreakPoint, decay,
+             confTarget, ">", 100.0 * successBreakPoint, decay,
```

Comment:
nit: could simply put the ">" directly into the format string instead of the "%s" and have one argument less

Reply:
Yeah right